internal/sbi/httpcallback/routers.go

```diff
@@ -100,4 +100,11 @@ var routes = Routes{
 		"/n1-message-notify",
 		HTTPN1MessageNotify,
 	},
+
+	{
+		"AmfHandleDeregistrationNotification",
+		strings.ToUpper("Post"),
+		"/amf-implicit-deregistration/3gpp-access/:ueid",
```

Comment:
modify as below:
```
/deregistration/:access/:ueid
```
add query parameter `type=implicit`

real URI will be like:
`{URL}/deregistration/3gpp/imsi-208930000000001?type=implicit`

Reply:
Sorry, I forgot the request body of the deregistration notification contains the access type.
Therefore, It should not put the access type into the URI.

Reply:
@YouShengLiu
Sorry, no need use `type` queryParam, amf can determine to do implicit dereg by deregReason in DeregistrationData.